dd-sdk-android-compose/src/main/kotlin/com/datadog/android/compose/ExperimentalTrackingApi.kt

```diff
@@ -0,0 +1,17 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.compose
+
+/**
+ * Marker for the experimental API.
+ */
+@RequiresOptIn(
+    message = "This is an experimental animation API for Datadog Tracking." +
```

Comment:
Is ** animation** the word we're looking for here? I'd say:
> This is an experimental tracking API…
or 
> This is an experimental Jetpack Compose observability API…

Reply:
oops, copy-paste issue.